way2sms-api.php

```diff
@@ -67,25 +77,18 @@ function sendWay2SMS($uid, $pwd, $phone, $msg)
     }
     $p = urlencode($p);
 
-    // Send SMS
-    curl_setopt($curl, CURLOPT_URL, 'http://site'.$autobalancer.'.way2sms.com/quicksms.action');
+    // Setup to send SMS
+    curl_setopt($curl, CURLOPT_URL, 'http://site'.$autobalancer.'.way2sms.com/smstoss.action');
```

Comment:
I tried that already. If I change it to get a random number lets say it gets anything between 10-19, it gets redirected to a different load balancer. And when I further access it at this step, the sessions gets invalid. What to do?